vavr/src/main/java/io/vavr/collection/Map.java

```diff
@@ -289,7 +289,7 @@ default boolean containsValue(V value) {
     default <U> Seq<U> flatMap(Function<? super Tuple2<K, V>, ? extends Iterable<? extends U>> mapper) {
         Objects.requireNonNull(mapper, "mapper is null");
         // don't remove cast, doesn't compile in Eclipse without it
-        return (Seq<U>) iterator().flatMap(mapper).toStream();
```

Comment:
I'm not sure that it is still important, but take into account comment about "don't remove cast..."

Reply:
damn it, that was not my intention :)

Reply:
Or if you're using Eclipse somehow, and it compiles, it's probably safe to remove.

Reply:
I checked 'cleanup' in idea and it removed the cast ... so for now I restored it.